packages/components/tests/spec/tooltip_spec.js

```diff
@@ -44,6 +44,35 @@ describe('Test tooltip', function() {
       expect(floating.classList.contains('bx--tooltip--shown')).toBe(true);
     });
 
+    it('Should apply focus to a child element upon opening', function() {
+      element.dispatchEvent(new CustomEvent('click', { bubbles: true }));
+      expect(document.activeElement.closest('.bx--tooltip')).not.toBe(null);
+    });
+
+    it('Should remain open upon focusing the content within the tooltip', function() {
+      element.dispatchEvent(new CustomEvent('click', { bubbles: true }));
+
+      const content = floating.querySelector('.bx--tooltip__content');
+      content.dispatchEvent(new CustomEvent('focus', { bubbles: true }));
+      expect(floating.classList.contains('bx--tooltip--shown')).toBe(true);
+    });
+
+    // Known bug: https://github.com/carbon-design-system/carbon/issues/3835
+    // it('Should hide the tooltip upon focusing an external element and focus moves to external element', function() {
+    //   element.dispatchEvent(new CustomEvent('click', { bubbles: true }));
```

Comment:
Given we seem to have no way to make this test case work, would we remove this?

Reply:
@asudoh this test does work but it fails because there is the known bug that I linked to in the comments which is why I commented it out. This is still a valid bug with tooltips but I feel like it's beyond the scope of the issue I'm trying to resolve with this PR.

Reply:
Probably we can keep just L60? My point is of reflecting our finding where firing `click` event cannot emulate the change in focus.

Reply:
I can remove the rest of the test if you'd like. For clarification, that `click` is to open the tooltip. The bug is when i apply the `:focus` to the input which does move the focus but to the wrong element (hence the bug)